configmodels/model_inventory.go

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2024 Canonical Ltd.
+
+package configmodels
+
+type Gnb struct {
+	GnbName string `json:"gnbName"`
```

Comment:
Since this is under `Gnb`, can't we simply use `Name` (instead of `GnbName`?

Reply:
I was about to simplify because on the NMS and the charm we already use just `name`